deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/mqtt/MqttConnection.java

```diff
@@ -34,11 +34,6 @@
 
     private MqttListener mqttListener;
 
-    private final ObjectLock openLock = new ObjectLock();
-
-    // paho mqtt only supports 10 messages in flight at the same time
-    static final int MAX_IN_FLIGHT_COUNT = 10;
```

Comment:
>MAX_IN_FLIGHT_COUNT [](start = 21, length = 19)

This value is still being used in Mqtt.java

Reply:
No, it's not. It is `MqttConnection` in device project not deps.